### PR TITLE
StreamSelectLoop: Fix erroneous zero-time sleep (2)

### PR DESCRIPTION
**Pull request by kaduev13**
Resolving https://github.com/reactphp/event-loop/issues/48.

### CODE REVIEW

File: tests/StreamSelectLoopTest.php [hunk 1]
```diff
@@ -145,4 +146,34 @@ protected function forkSendSignal($signal)
...
+    /**
+     * https://github.com/reactphp/event-loop/issues/48
+     *
+     * Tests that timer with very small interval uses at least 1 microsecond
+     * timeout.
+     */
+    public function testSmallTimerInterval()
+    {
+        /** @var StreamSelectLoop|\PHPUnit_Framework_MockObject_MockObject $loop */
+        $loop = $this->getMock(StreamSelectLoop::class, ['streamSelect']);
```
Comment (jsor) [inline tests/StreamSelectLoopTest.php]: `::class` requires PHP 5.5

Comment (kaduev13) [inline tests/StreamSelectLoopTest.php]: @jsor, sorry, yes, just updated PR.

**Comment by kaduev13** [comment]
@clue, @jsor, @WyriHaximus, thank you! :)

**Comment by jsor** [comment]
@clue Filed #94 to add this patch to the 0.4 branch

**Pull request merged**